slices/apt.yaml

```diff
@@ -0,0 +1,104 @@
+package: apt
+
+essential:
+  - apt_copyright
+
+slices:
+  config:
+    contents:
+      /etc/apt/apt.conf.d/01-vendor-ubuntu:
+      /etc/apt/apt.conf.d/01autoremove:
+      /etc/apt/auth.conf.d/:
+      /etc/apt/keyrings/:
+      /etc/apt/preferences.d/:
+      /etc/apt/sources.list.d/:
+      /etc/apt/trusted.gpg.d/:
+
+  apt-get:
+    essential:
+      - apt_config
+      - apt_default-source-list
+      - apt_var
+      - base-files_bin
+      - coreutils_bins
```

Comment:
see comments about `coreutils_bins` and `sed_bins` in my review of #617

Reply:
just noticed that this PR slices `apt-key`, which is a shell script, in addition to other `apt-...` bits which are bins. lemme investigate some more

Reply:
ok. looks like it dopes use `sed`, so nvm that

Reply:
and `apt-key` contains about 22 packages from `coreutils` so, while i would still advocate to subslice it, its fine as is.